Features/Health.cs

```diff
@@ -14,8 +18,19 @@ internal class Health : ToggleFeature
 
 		public override bool Enabled { get; set; } = false;
 
+		[ConfigurationProperty]
+		public bool VitalsOnly { get; set; } = false;
+
+		[ConfigurationProperty]
+		public bool RemoveNegativeEffects { get; set; } = true;
+
+		[ConfigurationProperty]
+		public bool FoodWater { get; set; } = true;
+
 		private static readonly Array _bodyParts = Enum.GetValues(typeof(EBodyPart));
 
+		private static String[] TargetBones = { "head", "spine3", "spine2", "spine1" };
+
 		[UsedImplicitly]
 		protected static bool ApplyDamagePrefix(object? __instance, ref float __result)
```

Comment:
Internally, `ActiveHealthController` is using the following signature:

```csharp
public float ApplyDamage(EBodyPart bodyPart, float damage, DamageInfo damageInfo);
```

See we use it as a postfix for hitmarkers: 
https://github.com/sailro/EscapeFromTarkov-Trainer/blob/master/Features/Hits.cs#L61

So let's rewrite the patch method from:

```csharp
protected static bool ApplyDamagePrefix(object? __instance, ref float __result)
```

to:

```csharp
protected static bool ApplyDamagePrefix(EBodyPart bodyPart, float damage, object damageInfo, object? __instance, ref float __result)
```

So you'll have access to the bodypart.

Reply:
Nope no clue